src/utils.py

```diff
@@ -233,4 +235,43 @@ def __getattr__(self, item):
         return getattr(self.inner, item)
 
 
-app = App()
\ No newline at end of file
+app = App()
+
+
+def walk(input_path):
+    """
+    helper function to yield folder's file content
+    Input:
+        input_path: the path of the folder
+    Output:
+        generator of files in directory tree
+    """
+    for p in Path(input_path).iterdir():
+        if p.is_dir():
+            yield from walk(p)
+            continue
+        yield p.resolve()
+
+
+def upload_experiment_to_s3(experiment_id: str,
```

Comment:
I don't have any comments on this without running. Does it run?

Also, could you help us understand the upload structure? Currently in the outputs dir I have the following:
outputs/2021-03-04/22-43-10/
and in it I have the *.pth and wandb dir

What would that path translate to?
exp_id/2021-03-04/22-43-10/*
or without the date?

Reply:
with the date, since `include_parent=True` is default
I've tried the code, seems to work (both on dummy and real experiment folder)